adapters/facebook.go

```diff
@@ -115,18 +136,28 @@ func (a *FacebookAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder
 		if len(s) != 2 {
 			return nil, fmt.Errorf("Invalid placementId param '%s'", params.PlacementId)
 		}
-		if fbReq.Site != nil {
-			fbReq.Site.Publisher = &openrtb.Publisher{ID: s[0]}
-		}
-		if fbReq.App != nil {
-			fbReq.App.Publisher = &openrtb.Publisher{ID: s[0]}
+		pubId := s[0]
+
+		// BANNER
+		fbReqB, err := a.MakeOpenRtbBidRequest(req, bidder, params.PlacementId, pbs.MEDIA_TYPE_BANNER, pubId, i)
+		if err == nil {
+			err = json.NewEncoder(&requests[reqIndex]).Encode(fbReqB)
+			reqIndex = reqIndex + 1
+			if err != nil {
+				return nil, err
+			}
 		}
-		fbReq.Imp[0].TagID = params.PlacementId
 
-		err = json.NewEncoder(&requests[i]).Encode(fbReq)
+		// VIDEO
+		fbReqV, err := a.MakeOpenRtbBidRequest(req, bidder, params.PlacementId, pbs.MEDIA_TYPE_BANNER, pubId, i)
 		if err != nil {
-			return nil, err
+			err = json.NewEncoder(&requests[reqIndex]).Encode(fbReqV)
+			reqIndex = reqIndex + 1
+			if err == nil {
```

Comment:
Same here, should it mirror logic for Banner above on line 146? If there is an error then we return early?

Reply:
Fixing